intermediate_notebooks/E2E/mortgage/mortgage_e2e.ipynb

```diff
@@ -119,23 +164,14 @@
    "outputs": [],
    "source": [
     "def initialize_rmm_pool():\n",
-    "    from librmm_cffi import librmm_config as rmm_cfg\n",
-    "\n",
-    "    rmm_cfg.use_pool_allocator = True\n",
-    "    #rmm_cfg.initial_pool_size = 2<<30 # set to 2GiB. Default is 1/2 total GPU memory\n",
-    "    import cudf\n",
-    "    return cudf.rmm.initialize()\n",
+    "    cudf.utils.utils._set_rmm_config(use_pool_allocator=True, initial_pool_size= None)\n",
```

Comment:
Can you do an if statement that checks `cudf.__version__` to determine which RMM API to use?

Perhaps that is a better addition to a "benchmark-only .py" script version of this notebook. Ideally we can run this against older and newer RAPIDS versions without users needing to make any changes.

Reply:
New update should work for older and newer versions of cudf